src/backtrace_support.rs

```diff
@@ -21,21 +21,31 @@ pub static ref WELL_KNOWN_SYS_MODULES: Vec<&'static str> = {
         "sentry_types::",
         // these are not modules but things like __rust_maybe_catch_panic
         "__rust_",
+        "___rust_",
```

Comment:
This sounds fishy :)

Reply:
tbh, I was wondering the same thing :) I remember inconsistencies between Rust versions and playing around with this. The entire list was added atomically to Sentry, so no idea where this occurred.